hw/bsp/at32f403a_407/boards/at_start_f403a/board.h

```diff
@@ -50,7 +50,8 @@
 // Button
 #define BUTTON_PORT           GPIOA
 #define BUTTON_PIN            GPIO_PINS_0
-#define BUTTON_STATE_ACTIVE   0
+#define BUTTON_PULL           GPIO_PULL_DOWN
```

Comment:
don't have the board to test, not sure if it works, best guess based on: https://github.com/RT-Thread-Studio/sdk-bsp-at32f403a-at-start/blob/main/documents/AT_START_F403A_V1.2_SCH.pdf